spec/fixtures/files/300x100.svg

```diff
@@ -0,0 +1,6 @@
+<svg width="300" height="100" xmlns="http://www.w3.org/2000/svg">
+    <path fill="#dedede" stroke="#555" stroke-width="2" d="M2 2h296v96H2z"/>
```

Comment:
I think you want 2 space indentation here instead of 4 spaces here? (that seems to be the convention in our other [html fixtures](https://github.com/forem/forem/blob/master/spec/fixtures/approvals/katextag/render/generates_katex_output.approved.html#L3))